README.md

```diff
@@ -115,6 +115,10 @@ See the [troubleshooting section](#troubleshooting) for more details about the r
 
 ![](images/validation-success.png)
 
+If you're using a non AWS S3 service, you will need to use a custom endpoint, use path style URLs and disable session tokens.  We recommond you consult the documentation of the service for the requirements
+
+![](images/custom-s3-service-configuration.png)
```

Comment:
preview FTR: https://github.com/christapley/artifact-manager-s3-plugin/blob/master/README.md#configuration

Reply:
I'm adding the new fields to the configuration section